person.rb

```diff
@@ -20,3 +26,6 @@ def is_of_age?
   end
   # rubocop:enable Naming/PredicateName
 end
+
+pers = Person.new(age: 10)
```

Comment:
- [ ] Just as mentioned by the previous reviewer, I am getting the very same error as him. Kindly look at why this is the case.

![image](https://user-images.githubusercontent.com/86318284/219795628-9e05314d-a49b-4f4e-9a19-050b8c6bde95.png)

Reply:
Hey  @Nemwel-Boniface . 
I provided a class main.rb that has the code to test. It has a few changes cause of linter errors but it functions identically.